### PR TITLE
communications - initial draft

### PR DESCRIPTION
**Pull request by woozyking**

### CODE REVIEW

**Comment by odesiuz** [comment]
This is a good documentation. It presents a suggestive good communication pattern in our environment rather than compulsive I suppose. Nice

File: communications.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+2. If this needs a certain person's or group's attention, elevate its priority by pinging them right below the message, or start a thread and ping them there.
+3. A rare but not impossible reactor-meltdown level of urgency, yell out in any synchronous manner and get people's attention.
+
+## Text by Default
+
+Before you decide to have a face-to-face or voice/video conference, take some time to organize your thoughts, and write them down. Revise it to be as concise as possible. If it's not urgently blocking all of your next tasks, asynchronously send this message.
+
+## Transparent by Default
+
+Don't be embarrassed about asking "stupid" questions. Try to always write your messages in a publically accessible (or at a bare minimum, team accessible) channel. This way, the information is preserved and become searchable. Loosely, it becomes a part of the knowledge base, aside from more formal documentation.
```
Comment (dilshank) [inline communications.md]: Use another word other than `stupid`

File: communications.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Budget Communication Time
+
+For asynchronous communications, try to allocate a relatively fixed schedule to initiate, check and respond to messages. To a certain extent, this means try to optimize for the whole block of contiguous time that suits better for our job, which has an innate quality of being a creative endeavor.
+
+For synchronous communications:
+
+1. Aggressively timebox them. If a meeting was set to be an hour-long, ask if it can be shortened to 30 minutes; if it was set to be 30 minutes long, ask if it can be shortened to 15 minutes...
+2. Write down the agenda, and prepare every participant.
+3. Instead of abstractly talk about a problem in a hollow boardroom, demonstrate the problem, perhaps on-site or screen-shared (which can be your local branch of a project that you're working on). Do not stretch beyond the agreed timebox, make a decision and move on.
```
Comment (dilshank) [inline communications.md]: We could also add:

4. Ensure there are clear and concise take-aways. Every meeting must have a clear outcome
5. One of the individuals attending should concisely summarize the meeting and key points of the meeting. These notes should be posted on an appropriate channel for both participants and anyone else who might be interested in the contents but couldn't attend/join the meeting.
6. Each meeting is the collective productivity value of all its participants. Consider each participant to be on average contributing $200/hr; it is always good to judge the need/value of each meeting against the economic value of the meeting (what other productive task could these participants be doing?)

File: communications.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## Minimize Processes and Tools
+
+Processes and tools are supposed to serve us and to save us from events that undermine our productivity or even help to boost it. But on many occasions, these processes and tools become a burden on the makers. Instead, they merely become things for backline delegators to flirt with, to feel good and feel important about themselves. Or they become a means for those that are not good or have no interest at their job to deviate from real tasks, to show that they have poured in endless hours on the planning and blueprints and thus worthy of the pretended job security.
+
+Thus processes such as SCRUM, and tools such as Slack, which originally set out to solve real problems, have become things that people criticize or even hate. Because maintaining discipline for every participant of a tool or a process is an astonishingly hard task.
```
Comment (dilshank) [inline communications.md]: Too much like opinion - maybe better for an accompanying blog post on dev-blog

File: communications.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+Everyone has a different way of working and thinking. This fact should be embraced, instead of introducing rules and processes to resist or neglect it.
+
+Therefore, try to minimize processes and tools. Before you demand for a tool that can nicely map out your product's user workflow, use a stack of post-it and some colored markers, map it out on a virtual or physical whiteboard; instead of putting the effort to shop for a progress tracking tool that can make you happy (or miserable) about yourself at the end of a working day, how about start with writing the todos on a piece of paper.
+
+It's no wonders that forward-thinking managers at larger organizations even dare to bypass settled processes, so that teams can install a newer version of an operating system that's necessary for development, instead of waiting for the process to eventually get the IT department to have it done, perhaps after 3 months.
```
Comment (dilshank) [inline communications.md]: Too much like opinion for `common` - shift to a blog post.

File: communications.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+## ~~"Inbox Zero~~ Everything Zero"
+
+A busy email inbox is not the only contributing factor to interruptions. Nowadays we have way too many channels of receiving information, by choice or not.
```
Comment (dilshank) [inline communications.md]: correction:
`by choice or otherwise.`

File: communications.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+So instead of achieving "Inbox Zero", try to be as close as possible to "Everything Zero". This means diligently clear or defer any information that is not work-related. Some platforms offer one simple toggle to enter the "Do Not Disturb" mode. Use it. Clear up your desktop, notification stacks, browser windows/tabs, etc. This helps to minimize distractions, so you can find what you need to focus on, faster.
```
Comment (dilshank) [inline communications.md]: Remove the "So".
Suggestion:
`Instead of striving to achieve "Inbox Zero",....`

File: communications.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+For synchronous communications:
+
+1. Aggressively timebox them. If a meeting was set to be an hour-long, ask if it can be shortened to 30 minutes; if it was set to be 30 minutes long, ask if it can be shortened to 15 minutes...
+2. Write down the agenda, and prepare every participant.
+3. Instead of abstractly talk about a problem in a hollow boardroom, demonstrate the problem, perhaps on-site or screen-shared (which can be your local branch of a project that you're working on). Do not stretch beyond the agreed timebox, make a decision and move on.
+
+## Minimize Processes and Tools
+
+Processes and tools are supposed to serve us and to save us from events that undermine our productivity or even help to boost it. But on many occasions, these processes and tools become a burden on the makers. Instead, they merely become things for backline delegators to flirt with, to feel good and feel important about themselves. Or they become a means for those that are not good or have no interest at their job to deviate from real tasks, to show that they have poured in endless hours on the planning and blueprints and thus worthy of the pretended job security.
```
Comment (dilshank) [inline communications.md]: Can we trim this for the `common` and move opinion (legit) to the dev blog?

**Pull request merged**